module-3/back-end/index.js

```diff
@@ -85,6 +87,60 @@ app.put('/updateDocumentData/:id', async (req, res) => {
   }
 })
 
+app.post('/createProject', async (req, res) => {
+  try {
+    project = {
+      title: req.body.title,
+      description: req.body.description
+    }
+    const result = await Project.create(project)
+    res.status(200).json({result: result})
+  }
+  catch(err) {
+    res.status(500).json({message: "Internal server error", error: err})
+  }
+})
+
+app.put('/updateProject/:id', async (req, res) => {
+  console.log('REQ', req.params);
+  try {
+    project = {
+      id: req.params.id,
+      title: req.body.title,
+      description: req.body.description,
+      updated_at: new Date
+    }
+    
+    const result = await Project.update(project)
+    
+    if (result) {
+      res.status(200).json({updated: result})
```

Comment:
No need to wrap it into `updated` key, it would be an extra work on a frontend to parse all possible variants of these keys that you have introduced, just return the result as is `.json(result)`